Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,346 @@
+                ) {
+                    Text("Observer Height")
+                } minimumValueLabel: {
+                    Text(heightRange.lowerBound, format: numberFormat)
+                } maximumValueLabel: {
+                    Text(heightRange.upperBound, format: numberFormat)
+                }
+                .listRowSeparator(.hidden, edges: .top)
+            }
+            .presentationDetents([.fraction(0.25)])
+            .navigationTitle("Settings")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        isPresented = false
+                    }
+                }
+            }
+        }
+    }
+}
+
+private extension ShowLineOfSightBetweenGeoelementsView {
+    /// View model responsible for setting up the 3D scene, graphics, animation,
+    /// and performing line of sight analysis between moving and static geoelements.
+    @MainActor
+    @Observable
+    final class Model {
+        /// Set of predefined waypoints for animating the taxi's movement.
+        private let points = [
+            Point(latitude: 40.748469, longitude: -73.984513),
+            Point(latitude: 40.747786, longitude: -73.985068),
+            Point(latitude: 40.747091, longitude: -73.983452),
+            Point(latitude: 40.747762, longitude: -73.982961)
+        ]
+        
+        /// Height of the observer in meters. Updates the observer graphic when changed.
+        var height = 20.0 {
+            didSet {
+                changeObserverHeight(height)
+            }
+        }
+        
+        private var frameIndex = 0
+        private let frameMax = 120
+        private var pointIndex = 0
+    
+        /// The 3D scene containing basemap, elevation, and building layers.
+        let scene: ArcGIS.Scene = {
+            // Creates a scene and set an initial viewpoint.
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            // Add base surface from elevation service.
+            let elevationSource = ArcGISTiledElevationSource(url: .elevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            var buildingLayer = ArcGISSceneLayer(url: .buildingsService)
+            scene.addOperationalLayer(buildingLayer)
+            let camera = Camera(
+                lookingAt: .observerPoint,
+                distance: 700.0,
+                heading: -30.0,
+                pitch: 45.0,
+                roll: 0.0
+            )
+            scene.initialViewpoint = Viewpoint(
+                boundingGeometry: .observerPoint,
+                camera: camera
+            )
+            return scene
+        }()
+        
+        /// Graphics overlay used to render the observer and target symbols.
+        let graphicsOverlay: GraphicsOverlay = {
+            let overlay = GraphicsOverlay()
+            let renderer = SimpleRenderer()
```

Comment:
```suggestion
            overlay.sceneProperties.surfacePlacement = .relative
            let renderer = SimpleRenderer()
```
After moving the statement above here, `SurfacePlacement` doesn't need to be initialized as an object and can be set directly.